marklogic-data-hub-central/ui/e2e/cypress/support/pages/browse.tsx

```diff
@@ -293,6 +293,7 @@ class BrowsePage {
   }
 
   clickColumnTitle(index: number) {
+    cy.wait(500);
```

Comment:
Same comment as earlier. Let's try to add waitUntil on the next get statement and see if it works, so that the explicit waits can be avoided.

Reply:
I had to add explicit wait here also to fix random failure. I did try to add waitUntil but it didn't help to fix random failure issue on DHS.